public/js/controllers/invitePlayers.js

```diff
@@ -14,7 +14,11 @@ angular.module('mean.system')
     $scope.searchHelper = '';
     var token = localStorage.getItem('#cfhetusertoken');
 
-
+    $scope.closeReusedModal = function () {
+      $('#reuse-modal').modal('close');
+      game.isFilledUp = null;
```

Comment:
Why null and not false?

Reply:
That is the function that Ikechukwu wrote. I just moved it to the invitePlayer controller since the test was not passing.